src/App.js

```diff
@@ -10,13 +10,25 @@ class App extends Component {
     super();
     this.state = {
       movies: movieData.movies,
-      individualMovie: null
+      individualMovie: null,
+      error: null
     }
   }
 
+  componentDidMount = () => {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+			.then((response) => response.json())
```

Comment:
I might add a little error handling for checking if `response.ok` is good first before parsing it.  If it isn't good, make sure to call `throw new Error()`!